### PR TITLE
shuffle feature

### PR DESCRIPTION
**Pull request by vu9596**
shuffle feature
https://edu-redmine.framgia.vn/issues/2623

### CODE REVIEW

**Comment by vu9596** [comment]
ready

**Comment by vu9596** [comment]
ready

File: app/src/main/res/layout/layout_play_info.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
     <TextView
         android:id="@+id/text_artist"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_height="20dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/layout_play_info.xml]: hardcode

File: app/src/main/res/drawable-hdpi/ic_shuffle_none.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    <path android:fillColor="#757575" android:pathData="M417.983,286.085c-2.286,-1.902 -4.572,-2.847 -6.858,-2.847c-2.662,0 -4.853,0.852 -6.563,2.566c-1.711,1.711 -2.566,3.901 -2.566,6.563v54.823h-73.091c-8.378,0 -16.132,-1.383 -23.271,-4.147c-7.139,-2.759 -13.135,-6.088 -17.987,-9.993c-4.849,-3.901 -9.705,-9.28 -14.558,-16.135c-4.856,-6.852 -8.713,-12.898 -11.567,-18.135c-2.852,-5.226 -6.373,-12.135 -10.561,-20.693c-14.655,34.259 -27.596,60.24 -38.828,77.943c5.137,7.422 10.467,14.037 15.987,19.838c5.518,5.804 10.754,10.896 15.702,15.276c4.949,4.374 10.564,8.186 16.844,11.416c6.283,3.237 11.8,5.948 16.563,8.139c4.771,2.189 10.76,3.949 17.99,5.283c7.231,1.328 13.322,2.334 18.271,2.991c4.948,0.664 11.707,1.143 20.272,1.431c8.562,0.287 15.51,0.376 20.834,0.287c5.335,-0.096 13.045,-0.198 23.134,-0.287c10.089,-0.093 18.179,-0.144 24.271,-0.144v54.819c0,2.474 0.903,4.616 2.71,6.423c1.807,1.808 3.949,2.711 6.42,2.711c2.669,0 4.859,-0.855 6.563,-2.566l91.365,-91.358c1.711,-1.718 2.566,-3.901 2.566,-6.57c0,-2.666 -0.855,-4.853 -2.573,-6.563L417.983,286.085z"/>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/drawable-hdpi/ic_shuffle_none.xml]: format lại flie này bằng tay

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by vu9596** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**